src/rest.ts

```diff
@@ -212,4 +212,15 @@ client.interceptors.request.use((config) => ({
   },
 }));
 
-export default publishRest;
+const user = () => dandiRest.user;
+const loggedIn = () => !!user();
+const insideIFrame = (): boolean => window.self !== window.top;
+const cookiesEnabled = (): boolean => navigator.cookieEnabled;
```

Comment:
Is https://github.com/dandi/dandiarchive/issues/909 going to work without these exports?

Reply:
They're still being exported, see lines 220-225.